packages/app/src/runner/SpecRunnerHeaderRunMode.vue

```diff
@@ -57,16 +56,10 @@
 <script lang="ts" setup>
 import { computed } from 'vue'
 import { useAutStore } from '../store'
-import type { EventManager } from './event-manager'
 import SpecRunnerDropdown from './SpecRunnerDropdown.vue'
 import { allBrowsersIcons } from '@packages/frontend-shared/src/assets/browserLogos'
 import { useAutHeader } from './useAutHeader'
 
-const props = defineProps<{
```

Comment:
Event Manager was only needed for the Selector Playground, and since that doesn't appear in run mode it was fine to remove it from the run mode header.

Passing the width used to be important to manage the appearance of the header, but now that we have changed the offset and allow scrolling, we are also able to drop this explicit setting of width.